buildkite.sh

```diff
@@ -27,7 +27,27 @@ steps:
     retry:
       automatic:
         limit: 2
+EOF
+
+case "${BUILDKITE_REPO:-x}" in
+    "https://github.com/dlang/dmd.git" | \
+    "https://github.com/dlang/druntime.git" | \
+    "https://github.com/dlang/phobos.git")
+
+cat << 'EOF'
+steps:
+  - command: |
+        make -f posix.mak style
```

Comment:
Yes, this is only run for the respective repository, but there's _zero_ that a PR to DMD will change the source code of Phobos and thus break it's style target (well, it could break the build of dscanner, but that's separately tested.)